tests/unit/index.ts

```diff
@@ -82,12 +82,37 @@ registerSuite({
 			assert.include(contents, `from 'sub/baz';`);
 		});
 	},
+	'test prefixing of absolute paths with options.name': function () {
+		// we reuse foo-directories from the directory stress test above.
```

Comment:
oh hm.  You mean make a new project in test/support/ for this?  I can totally do that.